SPECS/libreswan.spec

```diff
@@ -20,66 +26,81 @@
-Version: 3.26
-Release: %{?prever:0.}1.3%{?prever:.%{prever}}%{?dist}
+Version: 4.12
+Release: %{?prever:0.}2.1%{?prever:.%{prever}}%{?dist}.3
 License: GPLv2
 Url: https://libreswan.org/
+
 Source0: https://download.libreswan.org/%{?prever:with_development/}%{name}-%{version}%{?prever}.tar.gz
 %if 0%{with_cavstests}
 Source1: https://download.libreswan.org/cavs/ikev1_dsa.fax.bz2
 Source2: https://download.libreswan.org/cavs/ikev1_psk.fax.bz2
 Source3: https://download.libreswan.org/cavs/ikev2.fax.bz2
 %endif
 
-Patch1: libreswan-3.26-asn1-zu.patch
-
 Group: System Environment/Daemons
-BuildRequires: bison flex pkgconfig
-BuildRequires: gcc
-BuildRequires: systemd systemd-units systemd-devel
-Requires(post): coreutils bash systemd
-Requires(preun): systemd
-Requires(postun): systemd
 
-Conflicts: openswan < %{version}-%{release}
-Obsoletes: openswan < %{version}-%{release}
-Provides: openswan = %{version}-%{release}
-Provides: openswan-doc = %{version}-%{release}
+Patch1: libreswan-4.3-maintain-different-v1v2-split.patch
+Patch2: libreswan-3.32-1861360-nodefault-rsa-pss.patch
+Patch3: libreswan-4.1-maintain-obsolete-keywords.patch
+Patch6: libreswan-4.3-1934186-config.patch
+Patch7: libreswan-4.9-2176248-authby-rsasig.patch
+Patch8: libreswan-4.12-ikev2-auth-delete-state.patch
+
+# XCP-ng
+Patch1000: Fix-compilation-with-older-gcc.patch
 
-BuildRequires: pkgconfig hostname
-BuildRequires: nss-devel >= 3.16.1, nspr-devel
-BuildRequires: pam-devel
-BuildRequires: libevent-devel
-BuildRequires: unbound-devel >= 1.6.0-6 ldns-devel
-BuildRequires: libseccomp-devel
-BuildRequires: libselinux-devel
 BuildRequires: fipscheck-devel
 Requires: fipscheck%{_isa}
-Buildrequires: audit-libs-devel
 
+BuildRequires: audit-libs-devel
+BuildRequires: bison
+BuildRequires: curl-devel
+BuildRequires: flex
+BuildRequires: gcc make
+BuildRequires: ldns-devel
 BuildRequires: libcap-ng-devel
-BuildRequires: openldap-devel curl-devel
+BuildRequires: libevent-devel
+BuildRequires: libseccomp-devel
+BuildRequires: libselinux-devel
+BuildRequires: nspr-devel
+BuildRequires: nss-devel >= %{nss_version}
+BuildRequires: nss-tools
+BuildRequires: openldap-devel
+BuildRequires: pam-devel
+BuildRequires: pkgconfig
+BuildRequires: hostname
+BuildRequires: redhat-rpm-config
+BuildRequires: systemd-devel
+BuildRequires: unbound-devel >= %{unbound_version}
+BuildRequires: xmlto
 %if 0%{with_efence}
 BuildRequires: ElectricFence
 %endif
-BuildRequires: xmlto
-
-Requires: nss-tools, nss-softokn
 Requires: iproute >= 2.6.8
-Requires: unbound-libs >= 1.6.6
+Requires: nss >= %{nss_version}
+Requires: nss-softokn
+Requires: nss-tools
+Requires: unbound-libs >= %{unbound_version}
+Requires(post): bash
+Requires(post): coreutils
+Requires(post): systemd
+Requires(preun): systemd
+Requires(postun): systemd
 
 %description
-Libreswan is a free implementation of IPsec & IKE for Linux.  IPsec is
+Libreswan is a free implementation of IKE/IPsec for Linux.  IPsec is
```

Comment:
The description evolved upstream, removing "To build KLIPS, see the kmod-libreswan.spec file.". Is there any reason for keeping it in ours?

Reply:
Fixed.